lib/task-data/tasks/install-centos.js

```diff
@@ -12,12 +12,12 @@ module.exports = {
         profile: 'install-centos.ipxe',
         hostname: 'localhost',
         comport: 'ttyS0',
-        domain: 'rackhd.github.com',
+        domain: 'rackhd',
```

Comment:
This was suggested by @yyscamper. This is also intended to keep consistent with the install-ESXi job where domain is set to 'rackhd'. It is suggested that having github.com in the domain makes rackhd less visible. I am open to this change. Let me konw if you are preferring to have github.com in the domain.